docs/releases/1.4.txt

```diff
@@ -990,7 +990,7 @@ CSRF protection extended to PUT and DELETE
 Previously, Django's :doc:`CSRF protection </ref/csrf/>` provided
 protection only against POST requests. Since use of PUT and DELETE methods in
 AJAX applications is becoming more common, we now protect all methods not
-defined as safe by :rfc:`2616` -- i.e., we exempt GET, HEAD, OPTIONS and TRACE,
+defined as safe by :rfc:`9110` -- i.e., we exempt GET, HEAD, OPTIONS and TRACE,
```

Comment:
I'd revert all changes to release notes as they are confusing, e.g. RFC 9110 didn't exist in 2012 when Django 1.4 was released.

Reply:
Done!